### ISSUE TITLE
[BUG]  s6-rc-compile: fatal: invalid /etc/s6-overlay/s6-rc.d/init-crontabs-config/type: must be oneshot, longrun, or bundle

### ISSUE THREAD

**Issue by zazizou** [linuxserver/docker-nextcloud#360]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

On the latest docker image, I am observing this critical issue, with the container hanging on these two messages:

-  `s6-rc-compile: fatal: invalid /etc/s6-overlay/s6-rc.d/init-crontabs-config/type: must be oneshot, longrun, or bundle`
-  `s6-rc-compile: fatal: invalid /etc/s6-overlay/s6-rc.d/init-nextcloud-config/type: must be oneshot, longrun, or bundle
`
And the `init` never ends while the WebUI will issue` server internal error`.

### Expected Behavior

Should run without an error. Previous image does not have the problem.

### Steps To Reproduce

A `docker run -it lscr.io/linuxserver/nextcloud:latest bash ` will expose the issue

### Environment

```markdown
- OS:Archlinux
- How docker service was installed: by package manager
```

### CPU architecture

x86-64

### Docker creation

```bash
As recommended by this repo README,  with a compose file.
```

### Container logs

```bash
Not relevant
```

**Comment by j0nnymoe** [comment]
While I appreciate you might not think the container start up logs aren't relevant or providing your docker compose snippet. It would be helpful for you to provide it so we can attempt to reproduce this error.

Because our smoke tests currently shows it booting fine https://ci-tests.linuxserver.io/linuxserver/nextcloud/latest/index.html

**Comment by zazizou** [comment]
I have tried modifying the files by hand and restart the container, but still broken.

I went back for  `lscr.io/linuxserver/nextcloud:27.0.2-ls266`, and everything is working as expected

**Comment by zazizou** [comment]
> While I appreciate you might not think the container start up logs aren't relevant or providing your docker compose snippet. It would be helpful for you to provide it so we can attempt to reproduce this error.
> 
> Because our smoke tests currently shows it booting fine https://ci-tests.linuxserver.io/linuxserver/nextcloud/latest/index.html

My compose file  is a copy paste of the readme of this repo, apart for the data volumes.

for the logs, not to break my setup again, here are some hints

`
% sudo docker run  -it lscr.io/linuxserver/nextcloud:latest bash 
% sudo docker logs 1f051780569b 
s6-rc-compile: fatal: invalid /etc/s6-overlay/s6-rc.d/init-crontabs-config/type: must be oneshot, longrun, or bundle
`
As you can see nothing happens beyond this message

Now this what  should happen:
`
% sudo docker run  -it lscr.io/linuxserver/nextcloud:27.0.2-ls266 bash 
% sudo docker logs f289830be1a8 
[migrations] started
[migrations] 01-nginx-site-confs-default: executing...
[migrations] 01-nginx-site-confs-default: succeeded
[migrations] 02-default-location: executing...
grep: /config/nginx/site-confs/default.conf: No such file or directory
[migrations] 02-default-location: succeeded
[migrations] done
usermod: no changes
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗ 
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝ 

   Brought to you by linuxserver.io
───────────────────────────────────────

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    911
User GID:    911
───────────────────────────────────────

Setting resolver to  8.8.8.8 8.8.4.4
Setting worker_processes to 8
generating self-signed keys in /config/keys, you can replace these with your own keys if required
.+...+....+......+...........+......+....+...+.....+...+....+.....+....+..+.............+..+.........+++++++++++++++++++++++++++++++++++++++*...+.....+....+.....+.+...............+......+...+.....+.......+......+..+.+......+......+..+..........+++++++++++++++++++++++++++++++++++++++*.......+....++++++
...+++++++++++++++++++++++++++++++++++++++*..............+...............+.+++++++++++++++++++++++++++++++++++++++*......+......+..........+...+......+.....+...+......+....+.....+..........+...+..+.............+...+..+.......+.....+......+.+.....+.........+......+.+............+.....+.......+........+...+............+.+...........+....+.........+..+.+........+.........................+...........+....+......+...+.....+.....................+.+.....+.+...+.....+.+........+..........+...+.....++++++
-----
Initializing nextcloud 27.0.2.1 (this can take a while) ...
Setting permissions
New nextcloud instance
Please run the web-based installer on first connect!
Initializing finished
After completing the web-based installer, restart the Nextcloud container to apply default memory caching and transactional file locking configurations.
Alternatively, you can apply your own configurations by editing /config/www/nextcloud/config/config.php following the documentation:
https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/caching_configuration.html
https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/files_locking_transactional.html
# do daily/weekly/monthly maintenance
# min   hour    day     month   weekday command
*/15    *       *       *       *       run-parts /etc/periodic/15min
0       *       *       *       *       run-parts /etc/periodic/hourly
0       2       *       *       *       run-parts /etc/periodic/daily
0       3       *       *       6       run-parts /etc/periodic/weekly
0       5       1       *       *       run-parts /etc/periodic/monthly
# nextcloud cron
*/5     *       *       *       *       s6-setuidgid abc php -f /app/www/public/cron.php
cp: not replacing '/config/crontabs/root'
[custom-init] No custom files found, skipping...
crond[366]: crond (busybox 1.36.1) started, log level 5
[ls.io-init] done.

`

**Comment by drizuid** [comment]
What is the output of `id 1000 && ls -asnl /path/to/appdata`

**Comment by buckley310** [comment]
I ran into a similar issue on kubernetes/k3s using another linuxserver image this week. The old version worked but the latest version gave me this error. I found that `/etc/s6-overlay/s6-rc.d/ci-service-check/type` was an empty file on my k3s server, but not on my local machine. Manually deleting the image on the server and having it re-download fixed the issue. Not sure what the root cause was. I am reasonably sure the disk was never full.
